### DISCUSSION TITLE
docs/ninjutsu/low-ranking-ninja/packing-up-early-fr-work

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#5] [General]
# docs/low-ranking-ninja/packing-up-early-fr-work

看手机看着快下班了，就提前收拾东西，合理规划时间，不浪费一分一秒，倒数三二一卡点下班，结束元气满满的一天，成为一个自律的时间管理大师！

https://996.ninja/docs/low-ranking-ninja/packing-up-early-fr-work

**Comment by bobo20180714** [comment]
qw

**Comment by Pan-Binghong** [comment]
学到了

**Comment by 135246888** [comment]
可恶我们公司不让你准点下班

**Reply by MingGH** [reply]
呼吁大家一起准点下班🥸
